src/content/user-guide/arm-alpha.mdx

```diff
@@ -0,0 +1,213 @@
+YOLOv5 (Object Detection):
+```
+zoo:cv/detection/yolov5-s/pytorch/ultralytics/coco/base-none
+zoo:cv/detection/yolov5-s/pytorch/ultralytics/coco/pruned-aggressive_96
+zoo:cv/detection/yolov5-s/pytorch/ultralytics/coco/pruned_quant-aggressive_94
+zoo:cv/detection/yolov5-l/pytorch/ultralytics/coco/base-none
+zoo:cv/detection/yolov5-l/pytorch/ultralytics/coco/pruned-aggressive_98
+zoo:cv/detection/yolov5-l/pytorch/ultralytics/coco/pruned_quant-aggressive_95
+```
+
+YOLACT (Segmentation):
+```
+zoo:cv/segmentation/yolact-darknet53/pytorch/dbolya/coco/base-none
+zoo:cv/segmentation/yolact-darknet53/pytorch/dbolya/coco/pruned90-none
+zoo:cv/segmentation/yolact-darknet53/pytorch/dbolya/coco/pruned82_quant-none
+```
+
+BERT-base squad (Question Answering):
+```
+zoo:nlp/question_answering/bert-base/pytorch/huggingface/squad/base-none
+zoo:nlp/question_answering/bert-base/pytorch/huggingface/squad/12layer_pruned90-none
+zoo:nlp/question_answering/bert-base/pytorch/huggingface/squad/12layer_pruned80_quant-none-vnni
+zoo:nlp/question_answering/bert-base_cased/pytorch/huggingface/squad/base-none
+zoo:nlp/question_answering/bert-base_cased/pytorch/huggingface/squad/pruned90-none
+zoo:nlp/question_answering/bert-base_cased/pytorch/huggingface/squad/pruned80_quant-none-vnni
+```
+
+BERT-base sst2 (Sentiment Analysis):
+```
+zoo:nlp/sentiment_analysis/bert-base/pytorch/huggingface/sst2/base-none
+zoo:nlp/sentiment_analysis/bert-base/pytorch/huggingface/sst2/12layer_pruned90-none
+zoo:nlp/sentiment_analysis/bert-base/pytorch/huggingface/sst2/12layer_pruned80_quant-none-vnni
+zoo:nlp/sentiment_analysis/bert-base_cased/pytorch/huggingface/sst2/base-none 
+zoo:nlp/sentiment_analysis/bert-base_cased/pytorch/huggingface/sst2/pruned90-none
+zoo:nlp/sentiment_analysis/bert-base_cased/pytorch/huggingface/sst2/pruned90_quant-none
+```
+
+BERT-base conll2003 (Token Classification):
+```
+zoo:nlp/token_classification/bert-base/pytorch/huggingface/conll2003/base-none
+zoo:nlp/token_classification/bert-base/pytorch/huggingface/conll2003/12layer_pruned90-none
+zoo:nlp/token_classification/bert-base/pytorch/huggingface/conll2003/12layer_pruned80_quant-none-vnni
+```
+
+DistilBERT squad (Question Answering):
+```
+zoo:nlp/question_answering/distilbert-none/pytorch/huggingface/squad/base-none
+zoo:nlp/question_answering/distilbert-none/pytorch/huggingface/squad/pruned90-none
+zoo:nlp/question_answering/distilbert-none/pytorch/huggingface/squad/pruned80_quant-none-vnni
+```
+
+## Get Started
+
+Once you install the `deepsparse-nightly` package, the system will automatically detect that it is 
+running on ARM. Therefore, once you install with PyPI, you can use DeepSparse as usual.
+
+#### Benchmark Performance
+
+Let's benchmark DeepSparse's throughtput on BERT to ONNX Runtime's on an AWS `M6g.16xlarge` instance.
+
+Make sure you have ONNX Runtime installed:
+
+```
+pip install onnxruntime
+```
+
+At batch 64, ONNX runtime achieves 19 items/second running the standard BERT model.
```

Comment:
ONNX Runtime
Because Runtime has an initial cap in previous instances, I'm assuming it's a proper name and should have an initial cap here as well.